backend/stewards/models.py

```
@@ -13,11 +14,47 @@ class Steward(BaseModel):
+    steward = models.ForeignKey(
+        Steward,
+        on_delete=models.CASCADE,
+        related_name='feature_candidate_scores',
+    )
+    score = models.PositiveSmallIntegerField(
+        validators=[MinValueValidator(0), MaxValueValidator(3)],
+        help_text="0 = not interesting, 1 = weak, 2 = good, 3 = strong.",
+    )
+
+    class Meta:
+        unique_together = ['submission', 'steward']
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)